Packs/CommonScripts/TestPlaybooks/ParseWordDoc-Test.yml

```diff
@@ -0,0 +1,223 @@
+id: ParseWordDoc-Test
+version: -1
+name: ParseWordDoc-Test
+starttaskid: "0"
+tasks:
+  "0":
+    id: "0"
+    taskid: 3cc79ed6-9c1e-4a2f-85b7-8c93342416ac
+    type: start
+    task:
+      id: 3cc79ed6-9c1e-4a2f-85b7-8c93342416ac
+      version: -1
+      name: ""
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "1"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 50
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "1":
+    id: "1"
+    taskid: 9715984c-af56-4cbd-8c24-df9a41e237f3
+    type: regular
+    task:
+      id: 9715984c-af56-4cbd-8c24-df9a41e237f3
+      version: -1
+      name: Delete Context
+      description: Delete field from context
+      scriptName: DeleteContext
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "2"
+    scriptarguments:
+      all:
+        simple: "yes"
+    separatecontext: false
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 190
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "2":
+    id: "2"
+    taskid: d5a2a4e5-220c-4138-8fe4-98ca09c71e5a
+    type: regular
+    task:
+      id: d5a2a4e5-220c-4138-8fe4-98ca09c71e5a
+      version: -1
+      name: Download Word File
+      description: Sends http request. Returns the response as json.
+      scriptName: http
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "3"
+    scriptarguments:
+      filename:
+        simple: file-sample.doc
+      method:
+        simple: GET
+      saveAsFile:
+        simple: "yes"
+      url:
+        simple: https://file-examples-com.github.io/uploads/2017/02/file-sample_100kB.docx
```

Comment:
a bit concerned that we download a file that we dont control, lets use a file hosted in our repo